src/server/rest/v1/service/ChargingStationService.ts

```diff
@@ -1154,6 +1155,27 @@ export default class ChargingStationService {
     // Get the Charging station
     const chargingStation = await UtilsService.checkAndGetChargingStationAuthorization(
       req.tenant, req.user, filteredRequest.chargingStationID, action, null, { withSite: true, withSiteArea: true });
+    // Get the OCPP Client
+    const chargingStationClient = await ChargingStationClientFactory.getChargingStationClient(req.tenant, chargingStation);
```

Comment:
Why do you get the charging station client here?
I don't see where it is used by reading the code.

Reply:
Removed, this was copied from executeChargingStationCommand in a previous version :)